frontend/src/store/localStorage.js

```diff
@@ -104,6 +105,8 @@ const useLazyLocalStorage = () => {
 export const useLocalStorageStore = defineStore('localStorage', () => {
   const logger = useLogger()
 
+  const configStore = useConfigStore()
```

Comment:
The `localStorageStore` should not depend on the `configStore`. It is not the task of this store to do validations